app/models/school.rb

```diff
@@ -74,10 +74,6 @@ def not_registered?
     induction_coordinator_profiles.none?
   end
 
-  def partially_registered?
```

Comment:
I wonder if we should rename fully_registered? to registered? as we no longer have partially_registered